internal/api/registry/manifests.go

```diff
@@ -198,11 +198,14 @@ func (a *API) handleGetOrHeadManifest(w http.ResponseWriter, r *http.Request) {
 			`UPDATE manifests SET last_pulled_at = $1 WHERE repo_id = $2 AND digest = $3`,
 			a.timeNow(), dbManifest.RepositoryID, dbManifest.Digest,
 		)
-		if err != nil {
-			logg.Error(
-				"could not update last_pulled_at timestamp on manifest %s@%s: %s",
-				repo.FullName(), dbManifest.Digest, err.Error(),
-			)
+
+		if err == nil {
+			if dbManifest.LastPulledAt != nil && a.timeNow().Add(-7*24*time.Hour).Before(*dbManifest.LastPulledAt) {
+				logg.Info("last_pulled_at timestamp of manifest %s@%s got updated by more than 1 day by user %s",
```

Comment:
The message does not reflect the condition that was checked:
```suggestion
				logg.Info("last_pulled_at timestamp of manifest %s@%s got updated by more than 7 days by user %s",
```